htdocs/societe/class/api_thirdparties.class.php

```diff
@@ -1170,6 +1171,127 @@ function deleteCompanyBankAccount($id, $socid){
 		return $account->delete(DolibarrApiAccess::$user);
 	}
 
+	/**
+	 * Generate a sepamandate Document
+	 * 
+	 * @param int $socid thirdparty id
+	 * 
+	 * @return array Check success
+	 * 
+	 * @url GET /generateMandat/{socid}
+	 */
+	public function generateMandat($socid){
```

Comment:
Add here param companybankid  (see other comment later)...

Reply:
Ok, I'll do it